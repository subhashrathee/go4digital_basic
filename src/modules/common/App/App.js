import React from 'react'
import Header from '../Header/Header'
import Routes from '../Routes/Routes'

/**
* App class loads in the header and the routes used in the header.
*
* Props:      None
*
* State vars: None
*/
class App extends React.Component {
  render() {
      return (
        <div>
          <Header />
          <Routes />
        </div>
      )
  }
}

export default App
