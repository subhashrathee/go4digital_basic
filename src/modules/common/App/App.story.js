import React from 'react';
import App from './App'
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'

storiesOf('App', module)
  .add('Base App', () => (
    <BrowserRouter>
      <App />
    </BrowserRouter>
  ));
