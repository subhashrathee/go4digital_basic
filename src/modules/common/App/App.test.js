import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('App component', () => {
  const component = renderer.create(
    <BrowserRouter>
    <App />
    </BrowserRouter>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
