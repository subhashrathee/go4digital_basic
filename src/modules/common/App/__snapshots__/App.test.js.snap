// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component 1`] = `
<div>
  <header>
    <div
      id="wrap"
    >
      <a
        aria-current="true"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <img
          alt="Logo"
          className="logo"
          src="/img/logo.png"
        />
      </a>
      <nav
        id="nav"
      >
        <ul>
          <li>
            <a
              aria-current="true"
              className="selected"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </li>
          <li>
            <a
              aria-current={false}
              className={undefined}
              href="/services/"
              onClick={[Function]}
              style={undefined}
            >
              Services
            </a>
          </li>
          <li>
            <a
              aria-current={false}
              className={undefined}
              href="/about_us/"
              onClick={[Function]}
              style={undefined}
            >
              About
            </a>
          </li>
          <li>
            <a
              aria-current={false}
              className={undefined}
              href="/blog/"
              onClick={[Function]}
              style={undefined}
            >
              Blog
            </a>
          </li>
          <li>
            <a
              aria-current={false}
              className={undefined}
              href="/contact_us/"
              onClick={[Function]}
              style={undefined}
            >
              Contact
            </a>
          </li>
          <li>
            <a
              href="https://www.instagram.com/go4digital1/"
              rel="noopener noreferrer"
              target="_blank"
              title="Instagram"
            >
              <img
                alt="Instagram Icon"
                className="instagram-header-icon"
                src="/img/instagram-header-icon.png"
              />
            </a>
          </li>
          <li>
            <a
              href="https://www.facebook.com/Go4digital-964819073705597/"
              rel="noopener noreferrer"
              target="_blank"
              title="go4digital at Facebook"
            >
              <img
                alt="Facebook Icon"
                className="instagram-header-icon"
                src="/img/FB-f-Logo__blue_29.png"
              />
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <div>
    <div>
      <div
        className="hero-image-home"
      />
      <div
        className="main"
      >
        <h1>
          go4digital: digital journey together
        </h1>
        <p>
          Congratulations on deciding to start your business's digital journey. Believe me, you are not alone. Travel this digital journey together with us as we just embarked on it.
        </p>
        <p>
          First step in the journey is to get a website, we have got our domain name, website hosting already sorted, now working through to the look and feel and digital content of our website. We can also help you out to reach to the same stage before we can work together to establish our businesses in the digital world
        </p>
        <p>
          Next step in the journey is to share your digital website address with your existing clients so that they can reach you as per their convenience and need. Get your website up and running and start sharing your business's web address with your existing and potential customers. Feel free to reach out and learn more on this.
        </p>
        <p>
          After having a web address and catering for your existing loyal customers' convenience, now is the time to try and get some more business. Thumb rule is, your digital + social media reputation leads to new business opportunities. Wondering how can you do that? Embark on this journey and we will educate you about this and partner with you on establishing your business brand on digital landscape.
        </p>
        <p>
          Run digital campaigns to reach out thousands of customers looking for services you offer. Want to know how? Reach out and ask for campaign packages and choose right plan as per your budget and business needs
        </p>
      </div>
      <footer>
        <p>
          Â© 
          2018
           go4digital. All rights reserved.
        </p>
      </footer>
    </div>
  </div>
</div>
`;
